components/higher-order/with-state/index.js

```diff
@@ -31,12 +31,6 @@ function withState( initialState = {} ) {
 					/>
 				);
 			}
-		}
-
-		WrappedComponent.displayName = getWrapperDisplayName( WrappedComponent, 'state' );
-
-		return WrappedComponent;
-	};
+		};
+	} );
```

Comment:
Missing `'withState'` as `modifierName`.

Reply:
(amended & pushed)

Reply:
Thanks 👍

Reply:
I don't see amend?

Reply:
Oh. Maybe it got nuked in a subsequent rebase, @gziolo?

Reply:
Yes, I did something wrong :(
Do you have this branch locally with your commit?

Reply:
> Yes, I did something wrong

My fault for rebasing your commit. ;)

> Do you have this branch locally?

I do, the [rebase](https://gist.github.com/mcsf/cfb44ce3b5f3d9d39865c4d42a24184b) is `0ae3d2` and the [fix](https://gist.github.com/mcsf/0b28501c52fab9bb0b0a3f3c57a7ec4b) is `e3a846`.

Reply:
Fixes with https://github.com/WordPress/gutenberg/commit/66b226898e3edc74a2a553c56648ff277002c0bd, thanks!